Meta/Azure/Lagom.yml

```diff
@@ -103,6 +99,8 @@ jobs:
               -DBUILD_LAGOM=OFF \
               -DENABLE_LAGOM_CCACHE=ON \
               -DCMAKE_INSTALL_PREFIX=tool-install \
+              -DCMAKE_C_COMPILER=$(host-cc) \
+              -DCMAKE_CXX_COMPILER=$(host-cxx) \
```

Comment:
with this setup, we still get coverage of the Lagom Tools with gcc, which should catch any obvious AK/LibCore mistakes.